### DISCUSSION TITLE
Need help with adding announcement banner

### DISCUSSION THREAD

**Discussion by MGomezN** [executablebooks/meta#802] [Q&A] [answered]
Hello!
I followed instructions https://jupyterbook.org/en/latest/web/announcements.html, but I'm unable to generate the announcement.

- Could you please help me to check if I'm missing something in my _config.yml file? 

- I shared it as .txt because .yml is not allowed. 
- I tried to copy-paste the text here, but the indentation was modified, so I think the file is better.

Thanks!  (and sorry if this is not the proper way to ask. I'm learning)

[_config.txt](https://github.com/executablebooks/meta/files/9385860/_config.txt)

**Comment by lwasser** [comment, marked as answer]
hi! @MGomezN  is your book on github for us to look at the config there? your file seems ok but it should be _config.yml not .txt as far as i understand.

**Reply by MGomezN** [reply]
Thanks, @lwasser! Yes, the book is on GitHub https://github.com/fatiando/tutorials. But now the banner Is working! I appreciate your help.
